projects/observability/src/shared/dashboard/widgets/topology/topology-widget.model.ts

```diff
@@ -68,6 +69,14 @@ export class TopologyWidgetModel {
   })
   public enableBoxStyle?: boolean = false;
 
+  @ModelProperty({
+    key: 'layoutType',
+    displayName: 'Layout Type',
+    type: STRING_PROPERTY.type,
+    required: false
+  })
+  public layoutType?: TopologyLayoutType = TopologyLayoutType.CustomTreeLayout;
```

Comment:
What's the purpose of a feature flag if it's also a widget input? The latter seems better - but we certainly don't want both and having to reconcile the two.

Reply:
The FF is for this widget input `layoutType ` only. We'll just enable the FF for selected customers to see the graphLayout, rest will see the old CustomTreeLayout. Once we are sure with our layout, we'll enable it for the rest and remove the FF.